.github/workflows/ci.yml

```
@@ -0,0 +1,241 @@
+  pull-requests: write
+  issues: write
+
+env:
+  GO_VERSION: '1.24.2'
+  COVERAGE_THRESHOLD: 50
+
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)